lib/internal/util/debuglog.js

```diff
@@ -24,10 +25,10 @@ let testEnabled;
 function initializeDebugEnv(debugEnv) {
   debugImpls = ObjectCreate(null);
   if (debugEnv) {
-    debugEnv = debugEnv.replace(/[|\\{}()[\]^$+?.]/g, '\\$&')
-      .replace(/\*/g, '.*')
-      .replace(/,/g, '$|^')
-      .toUpperCase();
+    debugEnv = StringPrototypeReplace(debugEnv, /[|\\{}()[\]^$+?.]/g, '\\$&');
```

Comment:
`%String.prototype.replace%` looks up `Symbol.replace` of the regex.
```suggestion
    debugEnv = RegExpPrototypeReplace(/[|\\{}()[\]^$+?.]/g, debugEnv, '\\$&');
```

Reply:
`RegExpPrototypeReplace` is not available in Primordials

Reply:
Sorry, typed too fast
```suggestion
    debugEnv = RegExpPrototypeSymbolReplace(/[|\\{}()[\]^$+?.]/g, debugEnv, '\\$&');
```